hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/KeyDeletingService.java

```diff
@@ -309,6 +348,47 @@ private void processSnapshotDeepClean(int delCount)
                 RepeatedOmKeyInfo newRepeatedOmKeyInfo =
                     new RepeatedOmKeyInfo();
                 for (OmKeyInfo keyInfo : repeatedOmKeyInfo.getOmKeyInfoList()) {
+                  // To calculate Exclusive Size for current snapshot, Check
+                  // the next snapshot deletedTable if the deleted key is
+                  // referenced in current snapshot and not referenced in the
+                  // previous snapshot then that key is exclusive to the current
+                  // snapshot. Here since we are only iterating through
+                  // deletedTable we can check the previous and previous to
```

Comment:
nit: it would be great if you could clear out in comment that `rcCurrOmSnapshot` is next, `rcPrevOmSnapshot` is current and `rcPrevToPrevOmSnapshot` is previous as per the [doc](file:///Users/iamgroot/Downloads/Snapshot%20Size%20Calculation.pdf).

Reply:
May be:
```suggestion
                  // deletedTable we can check the previous and previous to
                  // previous snapshot to achieve the same.
                  // previousSnapshot - Snapshot for which exclusive size is
                  //                    getting calculating.
                  // currSnapshot - Snapshot's deletedTable is used to calculate
                  //                previousSnapshot snapshot's exclusive size.
                  // previousToPrevSnapshot - Snapshot which is used to check
                  //                if key is exclusive to previousSnapshot.
```

optional: you can also attach the design doc link for reference.